lib/virt_autotest/utils.pm

```diff
@@ -116,6 +116,13 @@ sub is_kvm_host {
     return check_var("SYSTEM_ROLE", "kvm") || check_var("HOST_HYPERVISOR", "kvm") || check_var("REGRESSION", "qemu-hypervisor");
 }
 
+#retrun 1 if libvirt 9.1+ is running
+sub is_libvirt91 {
+    unless (is_alp) {
+        return 1 if script_run('rpm -q libvirt-libs | grep "libs-9.[1-9]"') == 0;
```

Comment:
How about libvirt version 10.x?

Reply:
BTW, how about the function name is_libvirt91_plus?

Reply:
It is easier for checking libvirt9.x. It is possible this libvirt version checking would not been required any more for SLES when libvirt goes into 10.x.

Reply:
Yes, it is unknown for higher version behavior. We can change again when the code won't work again in the future by either a newer version or newer behavior. It's also ok to assume that higher version works the same with current latest version, just like you did for 9.[2-9] and other similar judgement of is_sle(15+). Up to you to decide :) .

How about function name? Will it be good to claim is_libvirt91 true for libvirt 9.2?

Reply:
Yes, I agree with is_moduler_libvirt(). And thinking of improving it a little bit and verifying on SLE.

Reply:
I am thinking of 
```
sub is_legacy_libvirtd {
    unless (is_alp) {
        return 1 if script_run('rpm -q libvirt-libs | grep -e "libs-9.0" -e "libs-[1-8]"') == 0;
    }
}
```
because modular libvirtd is the tendency and libvritd will be obsolete.